lib/phlex/view.rb

```diff
@@ -43,6 +44,16 @@ def call(buffer = +"", view_context: nil, parent: nil, &block)
 			buffer
 		end
 
+		def method_missing(name, *args, **kwargs, &block)
```

Comment:
`method_missing` and the `*args` and `**kwargs` splats are the slow bit. I wanted to use the `inherited` hook to eagerly define methods so `method_missing` wouldn't need to be hit.